app/components/house_keypad.js

```diff
@@ -109,25 +111,27 @@ var HouseKeypad = React.createClass({
     }
   },
   _toggleGarage: function(e) {
-    GarageDoorAPI.toggle();
+    this.props.GarageDoorAPI.toggle();
   },
   _off: function() {
-    AlarmAPI.off();
+    this.props.AlarmAPI.off();
```

Comment:
I'm not crazy about these `_ACTION` functions. If / when this becomes a real "component" these should be passed in as props.